doc/classes/GraphEdit.xml

```diff
@@ -233,6 +233,9 @@
 				Signal sent when user dragging connection from output port into empty space of the graph.
 			</description>
 		</signal>
+		<signal name="copy_nodes_request">
+			Signal sent when the user is pressed [code]CTRL + C[/code].
```

Comment:
Grammar: "when the user **presses**"
Style: I think `Ctrl + C` would be better (so lower case "trl").

Reply:
Fixed